ts/client/src/client.ts

```diff
@@ -1034,6 +1034,22 @@ export class MangoClient {
     return await this.sendAndConfirmTransactionForGroup(group, [ix]);
   }
 
+  public async sequenceCheck(
+    group: Group,
+    mangoAccount: MangoAccount,
+  ): Promise<MangoSignatureStatus> {
+    const ix = await this.program.methods
+      .sequenceCheck(mangoAccount.sequenceNumber)
+      .accounts({
+        group: group.publicKey,
+        account: mangoAccount.publicKey,
+        owner: (this.program.provider as AnchorProvider).wallet.publicKey,
+      })
+      .instruction();
+
+    return await this.sendAndConfirmTransactionForGroup(group, [ix]);
```

Comment:
this would send a transaction with a single sequenceCheck ix. Instead, make this a function that returns the sequence check instruction, so it can be composed with other ixes.